routing/index_router.cpp

```diff
@@ -63,14 +62,36 @@ SingleMwmRouter::SingleMwmRouter(string const & name, TCountryFileFn const & cou
   CHECK(m_directionsEngine, ());
 }
 
-IRouter::ResultCode SingleMwmRouter::CalculateRoute(m2::PointD const & startPoint,
-                                                    m2::PointD const & startDirection,
-                                                    m2::PointD const & finalPoint,
-                                                    RouterDelegate const & delegate, Route & route)
+IRouter::ResultCode IndexRouter::CalculateRoute(m2::PointD const & startPoint,
+                                                m2::PointD const & startDirection,
+                                                m2::PointD const & finalPoint,
+                                                RouterDelegate const & delegate, Route & route)
+{
+  string const startCountry = m_countryFileFn(startPoint);
```

Comment:
But do you plan to fix it? At least, leave a TODO and fill a jira ticket for that refactoring. Raw strings should not be used as map names, that's why CountryFile abstraction was implemented.